### PR TITLE
Illustrate how pull requests work

### PR DESCRIPTION
**Pull request by petroav**
Tutorial

### CODE REVIEW

**Comment by petroav** [comment]
@cawallin please review (don't actually)

File: README.md [hunk 1]
```diff
@@ -23,6 +23,18 @@ Comprehensive documentation can be found [here](http://prestodb.github.io/presto
...
+This is a test.
+
+
+
+
+
+
+
+
+I like turtles.
```
Comment (petroav) [inline README.md]: How dare you! I dislike turtles, remove this please.

Comment (cawallin) [inline README.md]: Can you change this to giraffes? I think that Lola would be sad if you favored turtles over giraffes.

**Comment by petroav** [comment]
Looks good Anton

**Comment by dklassen** [comment]
Please rebase against master

**Comment by ilfrin** [comment]
Couldn't you do it on a private repo? ;)

śr., 2 wrz 2015, 19:23 Anton Petrov użytkownik <notifications@github.com>
napisał:

**Pull request closed without merging**